certificate-service/app/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation project(path: ':pdfbox-generator')
```

Comment:
Move this to the top of dependencies. Makes it easier to see what dependencies we have between gradle-modules.